script/MintBPNori.s.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.15;
+
+import "forge-std/Script.sol";
+import "../contracts/BridgedPolygonNORI.sol";
+
+contract MintBPNori is Script {
+  /**
+   * @notice The BridgedPolygonNORI contract.
+   */
+  // address bpNoriAddress = 0x8D7713C76A2A6Ec517FB3C032524dd86425E24f4;
+  address bpNoriAddress = 0xBD16B6880F76CA8A3543dd7A2a60d375d0485dBB;
+  address defenderRelayer = 0x6Dc772F80495F47d8000530A59eE975B67B7c646;
+  BridgedPolygonNORI private _bridgedPolygonNORI;
+
+  function run() external {
+    vm.startBroadcast();
+    _bridgedPolygonNORI = BridgedPolygonNORI(bpNoriAddress);
+    _bridgedPolygonNORI.deposit(
+      defenderRelayer,
+      abi.encode(200000000000000000000)
+    ); // 200 NORI
+    vm.stopBroadcast();
+  }
+}
```

Comment:
this is the same as the following, right?

```bash
cast send \
  -c mumbai \
  --mnemonic-path $MNEMONIC_PATH \
  0xBD16B6880F76CA8A3543dd7A2a60d375d0485dBB \
  "deposit(address,bytes)" \
  0x6Dc772F80495F47d8000530A59eE975B67B7c646 \
  `cast --to-uint256 200000000000000000000` \
  /
```

Reply:
pretty sure it is... and I think using cast could make sense for something as simple as a deposit, but using a solidity script is going to make it so much simpler to handle something more complex and with much more complex arguments such as minting removals, especially if we want to make it a little bit easier to quickly parameterize, such as with the right number of different suppliers or removals or amounts.  running the solidity scripts is even easier than that cast command

Reply:
For sure, there's plentiful use cases for scripting, just want to make sure we're not rewriting their API just to constrain it. They do have a pretty robust cli and w/ a pretty nice DX (you can encode the structs via the cli as well for minting). 

Not sold on needing a script to batch mint to multiple suppliers for local use beyond seeding, but this is a useful example nonetheless! :100: